lib/iris/fileformats/_nc_load_rules/actions.py

```diff
@@ -43,11 +43,15 @@
 from functools import wraps
 import warnings
 
+from iris.config import get_logger
```

Comment:
Just wondering why this is introduced, as the "logger" references here on line 46, and 52 aren't in main, and I don't believe this file exists in the v3.0.x branch.

Reply:
Yup, good spot.

Basically, the latter commits https://github.com/SciTools/iris/pull/4276/commits/e7cb05027532b94cc46b6e7d56cb81f7e95fc1d1 and https://github.com/SciTools/iris/pull/4276/commits/611a5757b4cb39d45f6d28f5201388970188b8cc are there to deal with https://github.com/SciTools/iris/pull/4035/, which was merged to `v3.0.x` as part of the `v3.0.2` release.

Essentially, #4035 has some changes to the now re-implemented netCDF `pyke` rules (#4198), so I've had to incorporate that change into the new world.

As part of that, the netCDF load rules shouldn't be raising warnings now, as we're moving to `logging`.

Reply:
I understand now.  Thanks.